Restaurant page

MrSayyod opened the pull request:
- Not committed “console.log” into repo;
- Not used “Window.alert()” or “Window.confirm()”;
- Kept all the styling logic in CSS files;
- .lock files are in the repository to avoid problems with future versions of the dependencies;
- Used webpack;
- Included all webpack related packages in “devDependencies”;
- Added node_modules dir to .gitignore;
- set up linters;
- used 'development' branch as a default branch;
- committed changes often.

cyonii commented on src/index.js:
```diff
@@ -0,0 +1,49 @@
+import Content from './content';
+import Home from './home';
+// import menu from './menu'
+// import { doc } from "prettier";
```
- [ ] Please remove commented code. You can use git history if you need to reviwew previous versions of your codebase.

cyonii commented on src/menu.js:
```diff
@@ -0,0 +1,51 @@
+// import { node } from "webpack";
```
- [ ] No commented code

cyonii reviewed (changes requested):
### CHANGES REQUIRED ♻️✍️

#### TO HIGHLIGHT
- Descriptive PR description ✅
- Great project
- Clear commit messages ✅

#### TO IMPROVE
- [ ] The link to your license is broken, please fix it.
- [ ] I ran into an error when I run `npx run build` as you specified in your README.
- [ ] You should append HTML element to a `<div>` with `id=content` when the page is loaded (Use `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)). You have selected your content `<div>` using `getElementsByClassName` - which may not be unique in the DOM. Please use an ID selector.

Make the requested changes and resubmit for another review.
Good luck, and happy coding 🤓💻

> #### <br>Feel free to leave a comment or question if something is not clear.<br> Slack: @[CY Kalu](https://microverse-students.slack.com/team/U01494U96FR)<br><br>

MrSayyod commented:
- deleted comments;
- removed the broken link from the license in README;
- used 'container' div instead of 'content'. I don't think that the naming affects the working process of the project. Thanks

taiwo2 reviewed (changes requested):
# STATUS: CHANGES REQUIRED ♻️ 
@MrSayyod Great job for trying to implement suggested changes by the previous Code Reviewer, However changes are required
### Highlight
- [x] The link to your license . 👍🏻 
### To Improve
- [ ] I ran into an error when I run `npx run build` as you specified in your README.
- [ ] It appends HTML elements to the `<div>` with `id=content` when the page is loaded (Use `document.createElement` and `document.appendChild` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page))
-kindly use the naming specific by the project requirement `id= content`

### Once specific changes made kindly request for another review

MrSayyod commented:
- fixed 'npx run build' error;
- used 'content' name for container div;

mke2111 reviewed (approved):
# 😄 🥇  STATUS: APPROVED  🥇 👏🏽 

## Hey Happy Coder, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval. 👍🏽 

### **_Well Done_** 💯 

    1. The build command runs successfully.
    2. The UI is clean and user-friendly. 👍🏽 

### **_Room for Improvement_** 🤪 

 

- [x] The build command in the readme should be `npm run build` and not `nmp run build`

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

<hr>

### Best regards
Slack 🆘  Roy Mukuye
Twitter 🐦  [Roy Mukuye](https://twitter.com/Roymkenya)
LinkedIn 🔗  [Roy Mukuye](https://www.linkedin.com/in/roy-mukuye/)
